tools/cli/src/opt/kanidm.rs

```diff
@@ -1256,6 +1263,28 @@ pub enum Oauth2Opt {
     #[cfg(feature = "dev-oauth2-device-flow")]
     /// Disable OAuth2 Device Flow authentication
     DeviceFlowDisable(Named),
+    /// Rotate the signing and encryption keys used by this client. The rotation
+    /// will occur at the specified time, or immediately if the time is in the past.
+    /// Past signatures will continue to operate even after a rotation occurs. If you
+    /// have concerns a key is compromised, then you should revoke it instead.
+    #[clap(name = "rotate-cryptographic-keys")]
+    RotateCryptographicKeys {
```

Comment:
The help string here needs an example/explanation of what format the time needs to be in, and I'd suggest adding a `--now` flag that the client uses to set it to "now" when they invoke it.